controllers/market/updater.js

```diff
@@ -0,0 +1,157 @@
+const { Market, Warehouse, Bid, PlayerFantasy, Op } = require('../../databases/db');
+const responses = require('../../utils/responses/responses');
+
+const bidUpdate = async (req, res) => {
+    const userId = req.user.id.id;
+    const eventId = req.eventId;
+    const bidId  = req.params.bidId;
+    let { value, marketId, isDirectPurchase = false } = req.body;
+    value = parseInt(value);
+    if (value == null || marketId == null) {
+        return responses.errorDTOResponse(res, 400, 'Debe proporcionar un valor y un marketId');
+    }
```

Comment:
Creo que esto es inncesario, cuando es compra directa podemos omitir el value pt4
```suggestion
    if ((value == null && !isDirectPurchase) || marketId == null) {
        return responses.errorDTOResponse(res, 400, 'Debe proporcionar un valor y un marketId');
    }
```